babel/src/test/resources/sql/big-query.iq

```diff
@@ -1647,8 +1647,8 @@ FROM letters;
 +-----------+
 | example   |
 +-----------+
-| []        |
-| [A]       |
+| [     ]   |
```

Comment:
I'm not familiar with the binray type, but from the plan it makes sense.But the execution result is not what I expected and I don't know where BabelTest is executed.
```
LogicalProject(example=[SPLIT($0, X'ff':BINARY(1))])
  LogicalValues(tuples=[[{ X'0000000000' }, { X'4100000000' }, { X'42ff43ff44' }]])
```

Reply:
Let's merge https://github.com/apache/calcite/pull/4024 first, it actually takes care of some of these problems.
It addresses some very related issues.